icons/fire-alarm.json

```
@@ -0,0 +1,22 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "danielbayley"
+  ],
+  "tags": [
+    "flames",
+    "smoke",
+    "firefighter",
+    "fireman",
+    "department",
+    "brigade",
+    "station",
+    "emergency",
+    "safety"
+  ],
+  "categories": [
+    "devices",
+    "buildings",
+    "notifications"
```

`home` also fits doorbell.
Removing `buildings`, since that one is reserved for icons depicting or representing `buildings`.

```suggestion
    "devices",
    "notifications",
    "home"
```